### DISCUSSION TITLE
Huddle 2024-08-20

### DISCUSSION THREAD

**Discussion by stefaniebutland** [Openscapes/kyber#111] [General]
@seankross @jules32 @stefaniebutland @ateucher

**Comment by stefaniebutland** [comment]
More people contributing to kyber and we're talking about how we can extend it. 
- What workflows do we use? Merge all PRs? Ask for PR reviews? 
- Andy is willing to review PRs
- what's the scope of kyber

**Reply by stefaniebutland** [reply]
- developing R packages: whatever is on main is on  CRAN. For now, we're not putting in on CRAN
- but we can use R-universe, so having dev and main branches is good

For kyber, we do "cohort-driven-development". Can have a release prior to a cohort

**Reply by ateucher** [reply]
I think we are small and flexible enough to that we don't need to be too formal.  General flow:

- Open an issue, discuss and assign someone
- Open a PR, get at least one review - up to two - one for usability by the person who requested the feature, and one "engineering" review
- Open PRs against dev branch and when merge dev to main cut a release

**Reply by stefaniebutland** [reply]
Using kyber should feel like using `usethis`. Sean knows how he would do this

**Reply by stefaniebutland** [reply]
When Stef needs help on an issue, tag both Andy & Sean asking "are either of you able to help with this by DEADLINE". Then one of them self-assigns

**Reply by stefaniebutland** [reply]
We discussed approach to documentation - we have a long README that Stef typically updates; also Sean started creating Articles. Stef is happy to learn best/good enough practices for documenting in Articles

**Comment by ateucher** [comment]
Andy will work on #47 to set up an R-Universe and add kyber.

**Reply by stefaniebutland** [reply]
Let's plan to attend rOpenSci's next [Comm Call on R-universe](https://ropensci.org/commcalls/nov2024-r-universe/), Tuesday, 24 September 2024, 9am Pacific

**Comment by stefaniebutland** [comment]
Upcoming additions?
- CA Water Boards completion certificates w @AHolder1, final Cohort Call Oct 2, so deadline a week prior

**Reply by AHolder1** [reply]
FYI/update on my end - I am trying to find time to try and update the code to make certs that are custom for the Water Boards (because _of course_ I want to throw a wrench in things, ha!) - I am planning on diving into the code + testing locally on Sep 4 and was going to try and schedule a co-working sesh with @stefaniebutland once I have something to share. Would finding a time to connect that afternoon or shortly thereafter be helpful? I can pull up Stef's calendly and find something...

**Reply by stefaniebutland** [reply]
I love that you joined our mtg this way Anna

**Reply by stefaniebutland** [reply]
oh right, to get help with adding completion certificates, please open an issue and tag @ateucher to ask for help. He got our Champions certificates in "properly" based on my hack.

**Comment by stefaniebutland** [comment]
Build small modular things. Decided to keep Andy's grafana cost reporting in a separate package

**Reply by stefaniebutland** [reply]
related, from Sean: https://getwilds.org/sixtyfour/

**Comment by stefaniebutland** [comment]
Next steps:
Sean will address some issues and tag Stef when testing is needed.  
3 NOAA Fisheries Cohorts start Oct 8, so Stef will use kyber a lot in September
